cobspec/src/main/java/com/saralein/cobspec/controller/LogController.java

```diff
@@ -0,0 +1,58 @@
+package com.saralein.cobspec.controller;
+
+import com.saralein.cobspec.data.LogStore;
+import com.saralein.server.controller.Controller;
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Response;
+import java.util.Base64;
+
+public class LogController implements Controller {
+    private final LogStore logStore;
+    private final String username;
+    private final String password;
+
+    public LogController(LogStore logStore, String username, String password) {
+        this.username = username;
+        this.password = password;
+        this.logStore = logStore;
+    }
+
+    public Response respond(Request request) {
+        return isAuthorized(request) ? createAuthorizedResponse() : createUnAuthorizedResponse();
+    }
+
+    private Response createAuthorizedResponse() {
```

Comment:
Would the name of this method change if it didn't "new up" a response, but returned a response that was already constructed?

Reply:
Yes.  I started to change them to buildResponse, which isn't appropriate either.  Changed it to `getResponse` per our discussion.